index.html

```diff
@@ -2404,13 +2404,49 @@ <h2 id="similar-tools">Similar tools: tips &amp; tricks 🎩🐰</h2>
 
     <div class="well">
       <h2 id="cdda">CDDA (Audio CD) Ripping Tools</h2>
+      <!-- Find Drive Offset for Exact CD Ripping -->
+      <label class="recipe" for="find-offset">Find Drive Offset for Exact CD Ripping</label>
+      <input type="checkbox" id="find-offset">
+      <div class="hiding">
+        <h3>Find Drive Offset for Exact CD Ripping</h3>
+        <p>If you want to make CD rips that can be verified via checksums to other rips of the same content, you need to know the offset of your CD drive. Put simply, different models of CD drives have different offsets, meaning they start reading in slightly different locations. This must be compensated for in order for files created on different (model) drives to generate the same checksum. For a more detailed explanation of drive offsets see the explanation <a href="http://dbpoweramp.com/spoons-audio-guide-cd-ripping.htm">here.</a> In order to find your drive offset, first you will need to know exactly what model your drive is, then you can look it up in the list of drive offsets by Accurate Rip.</p>
+        <p>Often it can be difficult to tell what model your drive is simply by looking at it - it may be housed inside your computer or have external branding that is different from the actual drive manufacturer. For this reason, it can be useful to query your drive with CD ripping software in order to ID it. The following commands should give you a better idea of what drive you have.</p>
+        <p><strong>Cdda2wav:</strong> <code>cdda2wav -scanbus</code> or simply <code>cdda2wav</code></p>
+        <p><strong>CD Paranoia:</strong> <code>cdparanoia -vsQ</code></p>
+        <p>Once you have IDed your drive, you can search the <a href="http://www.accuraterip.com/driveoffsets.htm">Accurate Rip CD drive offset list</a> to find the correct offset for your drive as sourced by the community.</p>
+        <p><strong>Note:</strong> A very effective GUI based tool (macOS specific) for both for discovering drive offset as well as accurately ripping CDDAs is <a href="https://tmkk.undo.jp/xld/index_e.html">XLD</a>. Instructions for calibrating XLD can be found at <a href="https://wiki.hydrogenaud.io/index.php?title=XLD_Configuration">this page</a>.</p>
+        <p class="link"></p>
+      </div>
+      <!-- Find Drive Offset for Exact CD Ripping -->
+
+      <!-- Rip with CD Paranoia -->
+        <label class="recipe" for="cdparanoia">Rip a CD with CD Paranoia</label>
+        <input type="checkbox" id="cdparanoia">
+        <div class="hiding">
+          <h3>Rip a CD with CD Paranoia</h3>
+          <p><code>cdparanoia -L -B -O <em>[Drive Offset]</em> <em>[Starting Track Number]</em>-<em>[Ending Track Number]</em> <em>output_file.wav</em></code></p>
+          <p>This command will use CD Paranoia to rip a CD into separate tracks while compensating for the sample offset of the CD drive. (For more information about drive offset see <a href=#find-offset>the related ffmprovisr command.</a>)</p>
```

Comment:
I guess, the addresses should always be in quotation marks: `href="#find-offset"`.

Reply:
I am actually sure.